modules/twinkleprotect.js

```diff
@@ -517,6 +517,11 @@ Twinkle.protect.callback.changeAction = function twinkleprotectCallbackChangeAct
 				list: Twinkle.protect.protectionTags,
 				event: Twinkle.protect.formevents.tagtype
 			});
+
+			var isTemplateNamespace = mw.config.get('wgNamespaceNumber') === 10;
+			var isWikipediaNamespace = mw.config.get('wgNamespaceNumber') === mw.config.get('wgNamespaceIds').project;
+			var isArticlesForDeletion = mw.config.get('wgTitle').indexOf('Articles for deletion/') === 0;
+			var isCode = ['javascript', 'css', 'sanitized-css'].indexOf(mw.config.get('wgPageContentModel')) !== -1;
```

Comment:
shouldn't scribunto also come here?

Reply:
From my testing, I think the "Tagging options" part of the window is hidden when protecting a module. I could not get it to appear. From my testing, the act of protecting also doesn't modify any wikicode (i.e. no {{Pp-semi}} template is written anywhere).

Reply:
Ah yeah, the tagging options are [indeed disabled](https://github.com/wikimedia-gadgets/twinkle/blob/30ec330be93e84dffebc286484094bc8e66421cb/modules/twinkleprotect.js#L70) for Scribunto.